target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-c200-v1.dts

```diff
@@ -0,0 +1,379 @@
+	};
+
+	keys {
+		compatible = "gpio-keys";
+		pinctrl-0 = <&pmx_button_power &pmx_button_reset
+					&pmx_button_usb1 &pmx_button_usb2>;
+		pinctrl-names = "default";
+
+		power-button {
+			label = "Power Button";
+			linux,code = <KEY_POWER>;
+			gpios = <&gpio1 16 GPIO_ACTIVE_HIGH>;
+		};
+
+		reset-button {
+			label = "Reset Button";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio1 17 GPIO_ACTIVE_LOW>;
+		};
+
+		usb1 {
+			label = "USB1 Button";
+			linux,code = <BTN_0>;
+			gpios = <&gpio0 28 GPIO_ACTIVE_LOW>;
+		};
+
+		usb2 {
+			label = "USB2 Button";
+			linux,code = <BTN_1>;
+			gpios = <&gpio0 29 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio-poweroff {
+		compatible = "gpio-poweroff";
+		pinctrl-0 = <&pmx_poweroff>;
+		pinctrl-names = "default";
+		gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-0 = < &pmx_led_status_green &pmx_led_status_red
+				  &pmx_led_cloud_blue
+			      &pmx_led_disk1_green &pmx_led_disk1_red
+			      &pmx_led_disk2_green &pmx_led_disk2_red
+			      &pmx_led_disk_fail_green &pmx_led_disk_fail_red
+			      &pmx_led_usb1_green &pmx_led_usb1_red
+			      &pmx_led_usb2_green &pmx_led_usb2_red
+			      >;
+		pinctrl-names = "default";
+
+		led_status_green: status_green {
+			label = "green:status";
+			gpios = <&gpio1 7 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status_red: status_red {
+			label = "red:status";
+			gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
+		};
+
+		cloud_blue {
+			label = "blue:cloud";
+			gpios = <&gpio1 8 GPIO_ACTIVE_LOW>;
+		};
+
+		disk1_green {
+			label = "green:disk1";
+			gpios = <&gpio0 15 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "ata1";
+		};
+
+		disk1_red {
+			label = "red:disk1";
+			gpios = <&gpio0 17 GPIO_ACTIVE_LOW>;
+		};
+
+		disk2_green {
+			label = "green:disk2";
+			gpios = <&gpio0 16 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "ata2";
+		};
+
+		disk2_red {
+			label = "red:disk2";
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+		};
+
+		disk_fail_green {
+			label = "green:disk-fail";
+			gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
+		};
+
+		disk_fail_red {
+			label = "red:disk-fail";
+			gpios = <&gpio1 10 GPIO_ACTIVE_LOW>;
+		};
+
+		usb1_green {
+			label = "green:usb1";
+			gpios = <&gpio1 13 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "usbport";
+			trigger-sources = <&hub_port2>;
+		};
+
+		usb1_red {
+			label = "red:usb1";
+			gpios = <&gpio1 12 GPIO_ACTIVE_LOW>;
+		};
+
+		usb2_green {
+			label = "green:usb2";
+			gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "usbport";
+			trigger-sources = <&hub_port1>;
+		};
+
+		usb2_red {
+			label = "red:usb2";
+			gpios = <&gpio1 14 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&eth0 {
+	status = "okay";
+};
+
+&eth0port {
+	phy-handle = <&ethphy9>;
+};
+
+&i2c0 {
+	status = "okay";
+
+	rtc@30 {
+		compatible = "s35390a";
+		reg = <0x30>;
+	};
+
+	lm63@4c {
+		compatible = "national,lm63";
+		reg = <0x4c>;
+	};
+};
+
+&mdio {
+	status = "okay";
+	ethphy9: ethernet-phy@9 {
```

Comment:
Insert a newline between properties and subnodes.

Reply:
Done.